core/src/main/java/io/parsingdata/metal/Shorthand.java

```diff
@@ -136,10 +135,6 @@ private Shorthand() {}
     public static Token opt(final String name, final Token token) { return opt(name, token, null); }
     public static Token opt(final Token token, final Encoding encoding) { return opt(NO_NAME, token, encoding); }
     public static Token opt(final Token token) { return opt(token, null); }
-    public static Token nod(final String name, final ValueExpression size) { return new Nod(name, size, null); }
-    public static Token nod(final ValueExpression size) { return nod(NO_NAME, size); }
```

Comment:
I would opt to keep the nod shorthands. Def requires a name, which seems redundant for unknown data.